src/components/TableCellAvatar/Avatar.tsx

```diff
@@ -62,7 +64,11 @@ const Avatar: React.FC<AvatarProps> = ({
       })}
     >
       {badge}
-      {!thumbnail ? (
+      {!thumbnail && initials ? (
+        <MuiAvatar className={clsx(classes.avatar, avatarProps)}>
+          <Typography variant="h3">{initials}</Typography>
+        </MuiAvatar>
+      ) : !thumbnail ? (
```

Comment:
how about creating a new component that will conditionally render the proper avatar depending on given props?

Reply:
Goog point, fixed